src/medication-request/medication-request.resource.tsx

```diff
@@ -1,28 +1,24 @@
 import useSWR from "swr";
-import {
-  fhirBaseUrl,
-  formatDate,
-  openmrsFetch,
-  openmrsObservableFetch,
-  parseDate,
-} from "@openmrs/esm-framework";
+import { fhirBaseUrl, openmrsFetch, parseDate } from "@openmrs/esm-framework";
 import {
   AllergyIntoleranceResponse,
-  EncounterWithMedicationRequests,
-  EncountersWithMedicationRequestsResponse,
+  EncounterWithMedicationRequestsAndMedicationDispenses,
+  EncountersWithMedicationRequestsAndMedicationDispensesResponse,
```

Comment:
Names are getting longer 😅 

My only worry about this is that we might be coupling different types/tasks to to a single point. This isn't a blocker but definitely something that might need to be refactored later on.

Reply:
Yes, agreed @pirupius !  I actually wanted to start potentially refactoring the types into something clearer, but I figured I'd hold off a little bit while I learned more about how the code works, but you will likely see a renaming PR from me in the coming weeks!